docs/README.md

```diff
@@ -58,8 +58,6 @@ docker compose up
 
 To deploy your site using GitHub Pages you will need to add the [github-pages gem](https://github.com/github/pages-gem).
 
-> Note that GitHub Pages runs in `safe` mode and only allows a set of [whitelisted plugins](https://help.github.com/articles/configuring-jekyll-plugins/#default-plugins).
```

Comment:
This link was not working fine for the checker. But as this sentence does not add much value, I've deleted it.